sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("29.1.0.0")]
-[assembly: AssemblyFileVersion("29.0.0.0")]
+[assembly: AssemblyVersion("29.2.0.0")]
+[assembly: AssemblyFileVersion("29.2.0.0")]
```

Comment:
You don't have to update AssemblyFileVersion for minor version update.  Leave it to 29.0.0.0

Reply:
Fixed.

Reply:
@hyonholee @hapandar That is not correct. `AssemblyFileVersion` needs to be updated for every release, `AssemblyVersion` only when introducing a breaking change (aka a major version update).
So my suggestion is:
```suggestion
[assembly: AssemblyVersion("29.1.0.0")]
[assembly: AssemblyFileVersion("29.2.0.0")]
```
Read about it in our process document: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#signing-and-publishing-nuget-packges
An article explaining this: https://support.microsoft.com/en-us/help/556041

Reply:
@hapandar  Sorry, I was confused.  @isra-fel  is right.  Change only AssemblyFileVersion.

Reply:
Fixed